dask/dataframe/partitionquantiles.py

```diff
@@ -402,6 +402,7 @@ def percentiles_summary(df, num_old, num_new, upsample, state):
         each partition.  Use to improve accuracy.
     """
     from dask.array.percentile import _percentile
+    from ..utils import is_cupy_type
```

Comment:
Any reason not to have this import at the top level?

Reply:
No special reason here, I moved it to the top now and hopefully CI will pass this time, although it seems the failure was some sort of timeout https://github.com/dask/dask/pull/7325/checks?check_run_id=2041407897 .